core/Columns/Discriminator.php

```diff
@@ -46,7 +46,7 @@ public function __construct($table, $discriminatorColumn = null, $discriminatorV
     public function isValid()
     {
         return isset($this->discriminatorColumn)
-            && (is_numeric($this->discriminatorValue) || $this->discriminatorValue == ActionSiteSearch::CVAR_KEY_SEARCH_CATEGORY);
+            && (is_numeric($this->discriminatorValue));
```

Comment:
minor: the parentheses isn't needed anymore

Reply:
changed